tests/transform/image/test_color.py

```diff
@@ -13,6 +13,9 @@ def test_bgr2rgb():
     bgr_image = bgr2rgb(rgb_image, color_dim=0)
```

Comment:
`rgb_image = bgr2rgb(bgr_image, color_dim=0)`?

Reply:
Fixed in [36c845f](https://github.com/openxrlab/xrmocap/pull/78/commits/36c845f7f7918df025d92827c556e9892a88ddce).